src/Sfa.Das.Sas.Infrastructure/AssessmentOrgs/Models/OrganisationDocument.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Sfa.Das.Sas.Indexer.Infrastructure.AssessmentOrgs.Models
+{
+    public class OrganisationDocument
+    {
+        public string EpaOrganisationIdentifier { get; set; }
+
+        public string EpaOrganisation { get; set; }
+
+        public string OrganisationType { get; set; }
+
+        public string WebsiteLink { get; set; }
+
+        public string ContactAddress1 { get; set; }
```

Comment:
could you use 

        "primary" : "BCS, The Chartered Institute for IT",
        "secondary" : "Block D, North Star House",
        "street" : "North Star Avenue",
        "town" : "Swindon",